.circleci/config.yml

```diff
@@ -71,15 +71,22 @@ jobs:
   # Run parsing stats and publish them to the semgrep dashboard.
   autofix-printing-stats:
     docker:
-      - image: returntocorp/semgrep-dev:develop
+      - image: ocaml/opam:ubuntu
```

Comment:
you can add the comment of your PR here (why we don't use returntocorp/ocaml or returntocorp/semgrep-dev
docker images).